geonode_mapstore_client/client/js/plugins/Share.jsx

```diff
@@ -130,7 +131,8 @@ function Share({
                 </div>
                 <div className="gn-share-panel-body">
                     <SharePageLink url={pageUrl} label={<Message msgId="gnviewer.thisPage" />} />
-                    <SharePageLink url={embedUrl} label={<Message msgId={`gnhome.${resourceType}`} />} />
+                    <SharePageLink url={embedUrl} label={<Message msgId={`gnviewer.embed${resourceType}`} />} />
+                    {resourceType === 'document' && <SharePageLink url={downloadUrl} label={<Message msgId={`gnviewer.directLink`} />} />}
```

Comment:
This backport does not match the commit on master, please use the same commit on master

https://github.com/GeoNode/geonode-mapstore-client/commit/fe3ddfd828fca50afabc44ba2c377a691364e522#diff-e81b8603dbbf83db9f5f8a705cd5dafc164127477a90240b41072c740eb9e349R145